stdlib/public/Distributed/DistributedActorSystem.swift

```diff
@@ -12,28 +12,267 @@
+/// in order to write code that safely performs extra actions after it has fully initialized.
+///
+/// The ``actorReady(_)`` call on the actor system is a signal to the actor system that this actor _instance_ is now ready
+/// and may be resolved and interacted with via the actor system. Generally, a distributed actor system implementation
+/// will _weakly retain_ the actors it has readied, because retaining them strongly would mean that they will never be
+/// deallocated (and thus never resign their ID's).
+///
+/// > Note: Generally actor systems should retain actors _weakly_ in order to allow them be deinitialized when no longer in use.
+/// >
+/// > Sometimes though, it can be quite useful to have the system retain certain "well known" actors, for example when it is expected
+/// > that other nodes in the distributed system will need to interact with them, even if end-user code does no longer hold
+/// > strong references to them. An example of such "retain while actor system is active" distributed actors would be any kind
+/// > of actor which implements discovery or health check mechanisms between clustered nodes, sometimes called "system actors",
+/// i.e. actors that serve the actor system directly.
+///
+/// Next, we will discuss the the just mentioned resolve method, which is closely tied to readying actors.
+///
+/// ### Resolving (potentially remote) Distributed Actors
+///
+/// An important functionality of any distributed actor system is being able to turn a ``DistributedActor`` type and ``ActorID``
+/// into an reference to such actor, regardless where the actor is located. The ID should have enough information stored
+/// to be able to make the decision _where_ the actor is located, without having to contact remote nodes. Specifically,
+/// the implementation of ``DistributedActorSystem/resolve(id:as:)`` is _not_ `async` and should _not_ perform long running
+/// or blocking operations in order to return.
+///
+/// > Note: Currently only concrete distributed actors types can be resolved.
+///
+/// The actor system's ``DistributedActorSystem/resolve(id:as:)`` method is called by the compiler whenever end-users
+/// call the ``DistributedActor``'s ``resolve(id:using:)`` method. The return types of those methods differ,
+/// as the actor system's return type is `Act?` (and it may throw if unable to resolve the `ActorID`).
+///
+/// The actor system's `resolve` returning `nil` means that the ``ActorID`` passed to it refers to a _remote_
+/// distributed actor. The Swift runtime reacts to this by creating a remote actor reference (sometimes called a "proxy").
+///
+///
+/// ### Handling remote calls
+///
+/// Finally, calls on a _remote_ distributed actor reference's distributed methods are turned into invocations of
+/// `remoteCall(on:target:invocation:returning:throwing:)` (or `remoteCallVoid(on:target:invocation:throwing:)` for Void returning methods).
```

Comment:
```suggestion
/// ``remoteCall(on:target:invocation:returning:throwing:)`` (or ``remoteCallVoid(on:target:invocation:throwing:)`` for `Void` returning methods).
```

Reply:
Hah these sadly not... they're not "real methods" in source :'-( (Which is causing all kinds of pain, but yeah...